api_tests/registrations/views/test_registration_detail.py

```diff
@@ -435,15 +435,15 @@ def user_admin(self):
         return AuthUserFactory()
 
     @pytest.fixture()
-    def user_read_contrib(self):
+    def read_write_contrib(self):
```

Comment:
This was never a read contributor only in these tests to begin with, so just updating the name to match how it was being used.